samples/DotNet/Microsoft.Azure.ServiceBus/BasicDeadLetterQueueReceiver/Program.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace BasicDeadLetterQueueReceiver
+{
+    using Microsoft.Azure.ServiceBus;
+    using Microsoft.Azure.ServiceBus.Core;
+    using System;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    class Program
+    {
+        // Connection String for the namespace can be obtained from the Azure portal under the 
+        // 'Shared Access policies' section.
+        const string ServiceBusConnectionString = "{ServiceBus connection string}";
+        const string EntityPath = "{Queue Name or Subscription Path}";
+        static IMessageReceiver deadLetterReceiver;
+
+        static void Main(string[] args)
+        {
+            MainAsync().GetAwaiter().GetResult();
+        }
+
+        static async Task MainAsync()
+        {
+            deadLetterReceiver = new MessageReceiver(ServiceBusConnectionString, EntityNameHelper.FormatDeadLetterPath(EntityPath), ReceiveMode.PeekLock);
```

Comment:
>EntityNameHelper.FormatDeadLetterPath(EntityPath) [](start = 81, length = 49)

Since this is the primary focus of this sample (which can easily get lost between so many lines of code), probably we should mention this as comments on top of this line AND/OR mention it in readme.md